adapter/redisquota/redisquota.go

```diff
@@ -34,6 +35,11 @@ type redisQuota struct {
 	redisPool *connPool
 
 	redisError error
+
+	// algorithm for rate-limiting: either fixed-window or rolling-window.
+	// The fixed-window approach can allow 2x peak specified rate, whereas the rolling-window doesn't.
+	// Even though rolling-window is less lenient , it consumes more memory in redis.
```

Comment:
This comment should go in the proto so it'll be visible to people trying to configure the system.

The third line should say:

"The rolling-window algorithm's additional precision comes at the cost of increased redis resource usage."